src/scss/_responsive-scroll.scss

```diff
@@ -18,6 +31,15 @@
 				position: sticky;
 				left: 0;
 				background-color: _oTableGet('table-background');
+				&:after {
+					content: '';
+					position: absolute;
+					top: 0;
+					right: 0;
+					bottom: 0;
+					left: 0;
+					border-right: 2px solid _oTableGet('table-header-border-color');
+				}
```

Comment:
Clever solution. I reckon this might be working around a bug in Firefox/Safari/Chrome, maybe we should create issues on their bug trackers about this.